src/os/os_linux.c

```diff
@@ -62,3 +63,45 @@ void EmuX86Syscall(void* emu)
 {
     x86Syscall((x64emu_t*)emu);
 }
+
+void PersonalityAddrLimit32Bit(void)
+{
+    personality(ADDR_LIMIT_32BIT);
+}
+
+void ShowNativeBackTrace(int loglevel_min)
```

Comment:
can we don't have 2 functions calls for showBT (as they will appears on the backtrace)

Reply:
thanks for pointing this out, I converted it to a macro.